tests/test_algorithm.py

```diff
@@ -1965,3 +1966,44 @@ def analyze(context, perf):
                                 analyze=analyze)
         results = algo.run(self.panel)
         self.assertIs(results, self.perf_ref)
+
+
+class TestRemoveData(TestCase):
+    """
+    tests if futures data is removed after expiry
+    """
+    def setUp(self):
+        dt = pd.Timestamp('2015-01-02', tz='UTC')
+        env = TradingEnvironment()
+        ix = env.trading_days.get_loc(dt)
+
+        metadata = {0: {'symbol': 'X',
+                        'expiration_date': env.trading_days[ix+5],
```

Comment:
small style comment: could you put spaces around the binary operators to make it easier to read.